ssc/cmod_pvsamv1.cpp

```diff
@@ -984,6 +984,7 @@ void cm_pvsamv1::exec( ) throw (general_error)
 		else
 			b = Subarrays[nn]->selfShadingInputs.nmody * Subarrays[nn]->selfShadingInputs.width;
 		Subarrays[nn]->selfShadingInputs.row_space = b / Subarrays[nn]->groundCoverageRatio;
+		Subarrays[nn]->selfShadingSkyDiff.init(Subarrays[nn]->tiltDegrees, Subarrays[nn]->groundCoverageRatio);
```

Comment:
Curious if this could be included as part of a different function so users from other parts of the code don't have to remember to initialize it. Could it be part of the constructor?

Reply:
Very good point! Initialized in Subarray_IO::Subarray_IO